src/processor/map/map_copy_from.cpp

```diff
@@ -56,6 +56,12 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyFrom(LogicalOperator* logic
     }
 }
 
+static std::pair<column_id_t, bool> getWarningData(const BoundCopyFromInfo* copyFromInfo) {
```

Comment:
I guess u can remove this function now given its almost doing nothing.

Reply:
updated